core/tests/protractor_desktop/additionalEditorAndPlayerFeatures.js

```diff
@@ -87,6 +87,23 @@ describe('Full exploration editor', function() {
     explorationPlayerPage.reportExploration();
   });
 
+  it('should let learners suggest changes to an exploration', function() {
+    users.createUser('creator@editorAndPlayer.com', 'creatorEditorAndPlayer');
+    users.login('creator@editorAndPlayer.com');
+    workflow.createAndPublishExploration('Adding Fractions', 'Mathematics',
+      'Let us learn how to add fractions', 'English');
+    users.logout();
+
+    users.createUser('learner@editorAndPlayer.com', 'learner');
```

Comment:
@Showtim3 this user email and ID is used in a different test and I believe that is the reason why the tests fail on travis. 

When you tested this locally did you run just this test suite (using fit) or did you run the whole additionalEditorAndPlayerFeatures suite? If the former, can you please try the latter and see if it breaks locally too? (I am unable to run e2e tests on my local machine).

Reply:
@nithusha21  I used the fit method. Okay so you want me to run the additionalEditorAndPlayerFeatures suite? Okay I'll try that will post my results here.

Reply:
Yes, please run the entire test and post back the results here.

Reply:
It's behaving weird when i run the whole file but if i run a single test using `fit`, it passes.
![Screenshot 2019-08-26 at 11 38 58 PM](https://user-images.githubusercontent.com/30312043/63712186-c76e0980-c85a-11e9-9893-e1fa7aae01ec.png)

Reply:
@Showtim3, any updates on this?